Source/Core/Core/PowerPC/JitArm64/Jit.h

```diff
@@ -140,6 +140,9 @@ class JitArm64 : public JitBase, public Arm64Gen::ARM64CodeBlock, public CommonA
   void mtfsfx(UGeckoInstruction inst);
 
   // LoadStore
+  void WriteInitCacheLoop(const u32 cycle_count_per_loop, const Arm64Gen::ARM64Reg loop_counter,
```

Comment:
We normally don't put `const` for parameters in the header unless it has an actual effect, like a const reference or a pointer to const. (Feel free to keep the `const` in the .cpp file, though.)

This also applies to Jit64.